manifests/v/VirtusLab/ScalaCLI/v1.0.2/VirtusLab.ScalaCLI.locale.en-US.yaml

```
@@ -0,0 +1,79 @@
+# Created with WinGet Releaser v2 using Komac v1.8.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: VirtusLab.ScalaCLI
+PackageVersion: v1.0.2
+PackageLocale: en-US
+Publisher: Virtus Lab Sp. z o.o.

```

```suggestion
Publisher: scala-cli@virtuslab.com
```
Either this needs to match Control Panel, or the AppsAndFeaturesEntries need to be specified in the installer manifest